osfoffline/utils/__init__.py

```diff
@@ -1,6 +1,9 @@
 import hashlib
 import os
 
+from PyQt5 import QtCore
+from PyQt5.QtCore import QCoreApplication
+from PyQt5.QtWidgets import QApplication
```

Comment:
One other request: for unit testing purposes, we'd like to keep QT code out of the general utils module (see the CI server failure report for an example). 

Consider moving this decorator to somewhere more UI-centric, rather than mixing UI logic with application logic. You'll note we try to import QT in relatively few places throughout the codebase.

That should be it for drive-by comments from me; I now return you to your regular reviewers. :)